recipes/pycomlink/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pycomlink" %}
+{% set version = "0.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pycomlink/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 15b8b5dc4d5ad9e44e06baf58afbde00aa1432ae4fa697b3907df9cdce94a3da
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

Comment:
We can probably lighten the options passed here (`conda-build` handles many of these for us, except `-v`).

Reply:
Thanks for the quick review.

I took these from another package which has similar dependencies to mine to be on the save side.

I will go back to the suggested pip options from the example.

Reply:
Of course. 😄

Yep, that's generally a good idea. The docs and examples here are likely more up-to-date though. For things not included in either, taking things from other recipes is a good plan. 👍